i18n/locales/en.ts

```diff
@@ -65,6 +65,25 @@ const translations = {
     },
 
     satoshi_needs_you: 'Now be quick, Satoshi needs you.',
+    privacy: {
+      title: 'Our commitment to privacy',
+      paragraph_one:
+        'At Saving Satoshi, we are deeply committed to ensuring the privacy and security of our users. We believe in transparency and want you to understand how and why we track app usage. To achieve this, we use a powerful tool called <Link href="https://umami.is/" className="underline" target="_blank">Umami</Link>.',
+      sub_heading_one: 'What is Umami?',
+      paragraph_two:
+        'Umami is an open-source analytics platform that helps us gather essential insights about how you use our app. It allows us to make data-driven decisions and continuously enhance our product to better meet your needs.',
+      sub_heading_two: 'What Data Do We Collect?',
+      paragraph_three: `Rest assured, we only collect non-personal and anonymized data, such as: 
+      <ol>
+      <li><span className="font-bold">Product Improvement:</span> We use the data to identify areas where our app can be improved. By understanding how you use our app, we can make it more efficient, user-friendly, and secure.</li>
+      <li><span className="font-bold">Compatibility:</span>Understanding the devices and platforms you use allows us to optimize our app for various configurations, ensuring it works seamlessly for you.</li>
+      </ol>`,
```

Comment:
It also really doesn't seem to play nivcely with having these line breaks, it works better if there are no line-breaks